pkg/controller/controller_binding_test.go

```diff
@@ -591,6 +603,9 @@ func TestReconcileServiceBindingNonbindableClusterServiceClassBindablePlan(t *te
 			ExternalID:         testServiceBindingGUID,
 			SecretName:         testServiceBindingSecretName,
 		},
+		Status: v1beta1.ServiceBindingStatus{
+			UnbindStatus: v1beta1.ServiceBindingUnbindStatusRequired,
```

Comment:
This should be NotRequired. The test should be testing a new ServiceBinding for which no broker request has yet been made.

Reply:
Done.